index.js

```diff
@@ -0,0 +1,10 @@
+module.exports = function (input, context) {
+    //context.log("Node.js script processed queue message '" + json + "'");
```

Comment:
please remove the commented out lines - these samples are intended to be clean examples.